CMakeLists.txt

```diff
@@ -337,6 +337,24 @@ IF(NOT CYCLUS_DOC_ONLY)
                         OUTPUT_VARIABLE core_version
                         OUTPUT_STRIP_TRAILING_WHITESPACE)
     ENDIF(NOT "${CORE_VERSION}" STREQUAL "")
+
```

Comment:
This also should be a clean PR. Right now, it's stacked on top of the changes intended for issue 1769.